go/cmd/ct-fetch/ct-fetch.go

```diff
@@ -504,7 +504,9 @@ func (ld *LogSyncEngine) NewLogWorker(ctx context.Context, ctLogMeta *types.CTLo
 	}
 
 	metricKey := ctLogMeta.MetricKey()
-	metrics.SetGauge([]string{metricKey, "coverage"}, float32(logObj.MaxEntry-logObj.MinEntry+1)/float32(sth.TreeSize))
+	if sth != nil {
```

Comment:
Don't we already dereference `sth` on e.g. line 478?

Reply:
Ohh, right, not if `fetchErr` is something...

Reply:
Right, `sth` is only `nil` if `fetchErr` is not.